webrtc.html

```diff
@@ -3445,62 +3445,68 @@
         </ol>
         <p>To <dfn>check if negotiation is needed</dfn> for
         <var>connection</var>, perform the following checks:</p>
-        <ul>
+        <ol>
           <li>
             <p>If any implementation-specific negotiation is required, as
             described at the start of this section, return <code>true</code>.
             </p>
           </li>
+            <li>
+              <p>Let <var>description</var> be <var>connection</var>'s
+              <a data-for="RTCPeerConnection"
+              ><code>currentLocalDescription</code></a>.</p>
+            </li>
           <li>
             <p>If <var>connection</var> has created any
-            <code><a>RTCDataChannel</a></code>s, and no m= section has been
-            negotiated yet for data, return <code>true</code>.</p>
+            <code><a>RTCDataChannel</a></code>s, and no m= section in
+            <var>description</var> has been negotiated yet for data, return
+            <code>true</code>.</p>
           </li>
           <li>
-            <p>For each transceiver <var>t</var> in <var>connection</var>'s
+            <p>For each <var>transceiver</var> in <var>connection</var>'s
             <a>set of transceivers</a>, perform the following checks:</p>
-            <ul>
+            <ol>
               <li>
-                <p>If <var>t</var> isn't <a data-for="RTCRtpTransceiver">
-                stopped</a> and isn't yet <a>associated</a> with an m= section,
-                return <code>true</code>.</p>
+                <p>If <var>transceiver</var> isn't <a data-for="RTCRtpTransceiver">
+                stopped</a> and isn't yet <a>associated</a> with an m= section
+                in <var>description</var>, return <code>true</code>.</p>
```

Comment:
By the way, it now may be redundant to say "in description" here, since we define what an "associated" transceiver is independently. But I don't see that it hurts.

Reply:
I am thinking whether there are transceivers associated with session descriptions other than `currentLocalDescription`, e.g. `pendingLocalDescription`. Should the result be true or false in such case?

Reply:
These steps are only run in the `"stable"` signaling state, where there won't be any `pendingLocalDescription`, so that's not an issue.